DarkUI/DarkUI/Controls/DarkTreeView.cs

```diff
@@ -118,6 +118,14 @@ public int Indent
             }
         }
 
+        [Category("Appearance")]
```

Comment:
This implementation is different, because if it is unset, it doesn't override it, but uses the default values. I already tested it with brightness settings and it works perfectly fine.